public/css/toggle.css

```diff
@@ -53,15 +53,15 @@
 }
 
 .event-card {
-  transform: translate(110%, 30%);
+  transform: translate(4%, 15%);
```

Comment:
Don't use `translate` for `event-card` and `more-info` – the way you're using it, this looks like a job for `margin` or `padding`